app/src/main/java/org/dhis2/utils/reporting/CrashReportControllerImpl.kt

```diff
@@ -21,6 +22,14 @@ class CrashReportControllerImpl @Inject constructor() : CrashReportController {
         }
     }
 
+    override fun trackError(exception: Exception, message: String) {
+        val breadcrumb = Breadcrumb()
+        breadcrumb.type = "Info"
+        breadcrumb.message = message
+        Sentry.addBreadcrumb(breadcrumb)
+        Sentry.captureException(exception)
+    }
+
     override fun logException(exception: Exception) {
 //        Sentry.captureException(exception)
```

Comment:
??

Reply:
this was commented time ago to avoid sending non fatals to Sentry. I would create an issue to remove the method